### ISSUE TITLE
The /api/vid route no longer returns correct data

### ISSUE THREAD

**Issue by blubbll** [iv-org/invidious#1050]
Views and description are 0 / empty

**Comment by omarroth** [comment]
Mind providing an example?

**Comment by blubbll** [comment]
it only happens when i proxy the request via node-request to invidio.us. it works correctly with the invidious instance at invidious.snopyta.org.
i would love to host my own instance, but don't understand much about crystal, so i just let my server forward the api calls to the main instance.

**Comment by omarroth** [comment]
Likely duplicate of #1039. Is there a reason you're using `region` for all requests?

**Comment by blubbll** [comment]
> Likely duplicate of #1039. Is there a specific reason you're using `&region=` for all requests?

It would return the most relatable searches. I guess this is negative because for some countries there isnt enough (unblocked) proxies huh?
